test/e2e/toolchaincluster_test.go

```diff
@@ -24,13 +24,13 @@ func TestToolchainClusterE2E(t *testing.T) {
 	memberAwait := awaitilities.Member1()
 	memberAwait.WaitForToolchainClusterResources(t)
 
-	verifyToolchainCluster(t, hostAwait.Awaitility, memberAwait.Awaitility)
-	verifyToolchainCluster(t, memberAwait.Awaitility, hostAwait.Awaitility)
+	verifyToolchainCluster(t, hostAwait.Awaitility, memberAwait.Awaitility, true)
+	verifyToolchainCluster(t, memberAwait.Awaitility, hostAwait.Awaitility, false)
 }
 
 // verifyToolchainCluster verifies existence and correct conditions of ToolchainCluster CRD
 // in the target cluster type operator
-func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wait.Awaitility) {
+func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wait.Awaitility, hostoperator bool) {
```

Comment:
two things:
* it's just a detail, but it would make more sense to name the variable for example `singleCondition` or `onlyReadyCond`
* as I mentioned in the other PR, keep the naming convention and use camelCase for variable/field/param names

Reply:
okay i can update that, since it was temporary i though to use the operator name. and will remember to use Camel case for all